packages/form-core/src/choice-group/ChoiceGroupMixin.js

```diff
@@ -44,7 +44,12 @@ const ChoiceGroupMixinImplementation = superclass =>
        * @param {ChoiceInputHost} el
        * @param {any} val
        */
-      const checkCondition = (el, val) => el.choiceValue === val;
+      const checkCondition = (el, val) => {
+        if (typeof el.choiceValue === 'object') {
```

Comment:
When is this an object again? In our extension layer SelectRich?

Reply:
Yep :) our select rich extension indeed!